src/main/java/uk/gov/hmcts/reform/sscs/service/NotificationService.java

```diff
@@ -286,11 +286,12 @@ private String getSubscriptionDetails(String newSubscription, String oldSubscrip
 
     private boolean isEventAllowedToProceedWithValidData(NotificationWrapper notificationWrapper,
                                                          NotificationEventType notificationType) {
+        //FIXME: Can remove this block??
```

Comment:
looks like this code block can be removed judging by the other PR's

Reply:
Actually, speaking to Sheila, I need to keep this block. I'll remove the Fix